src/doom/m_random.c

```diff
@@ -63,3 +63,10 @@ void M_ClearRandom (void)
 {
     rndindex = prndindex = 0;
 }
+
+// inspired by the same routine in Eternity, thanks haleyjd
+int P_SubRandom (void)
+{
+    int r = P_Random();
```

Comment:
Am Samstag, den 23.07.2016, 11:38 -0700 schrieb Simon Howard:
> In case you aren't familiar with them, you'll want to read up on
> sequence points for some context on why this change is necessary.

Thanks, this was very interesting!

 - Fabian